### ISSUE TITLE
IPython should not assume jQuery for Javascript display

### ISSUE THREAD

**Issue by ian-r-rose** [ipython/ipython#10368]
Cross-posting this with jupyterlab/jupyterlab#1818.

The current implementation of the IPython Javascript display class assumes that jQuery is installed for loading external libs and css. This is not necessarily true on frontends (it is not true for JupyterLab, nor for nteract, as far as I know).

Most participants in the other issue seem to agree that this dependency should not be assumed (and possibly that the Javascript repr should be removed all together?).

**Comment by takluyver** [comment]
> and possibly that the Javascript repr should be removed all together?

That sounds like it would break a lot of notebooks that have used it.

**Comment by ian-r-rose** [comment]
Yes, I am not advocating doing that any time soon--just flagging that discussion over here.
